osu.Game.Rulesets.Mania.Tests/Mods/TestSceneManiaModDoubleTime.cs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using NUnit.Framework;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets.Mania.Mods;
+using osu.Game.Rulesets.Mania.Objects;
+using osu.Game.Rulesets.Mania.Replays;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.Replays;
+using osu.Game.Tests.Visual;
+
+namespace osu.Game.Rulesets.Mania.Tests.Mods
+{
+    public partial class TestSceneManiaModDoubleTime : ModTestScene
+    {
+        private const double offset = 18;
+
+        protected override Ruleset CreatePlayerRuleset() => new ManiaRuleset();
+
+        [Test]
+        public void TestHitWindowWithoutDoubleTime() => CreateModTest(new ModTestData
+        {
+            Mod = new ModNoMod(),
```

Comment:
This is slightly... unexpected if anything? I'd accept just... nothing, as in no mods set, as in

```suggestion
```

Does this do anything? It's probably fine either way, just a smidge weirded out...

Reply:
I don't think this test scene handles a `null` value... I didn't test it so that's just from looking at the code.

Reply:
It... doesn't? Seems fine to me in testing. Also this:

https://github.com/ppy/osu/blob/a8fc17188f9198024a88de4b0239894d0c63f575/osu.Game/Tests/Visual/ModTestScene.cs#L49-L50
https://github.com/ppy/osu/blob/a8fc17188f9198024a88de4b0239894d0c63f575/osu.Game/Tests/Visual/ModTestScene.cs#L118-L130

Reply:
Makes senset. It would only fail if setting `Mod = null`